src/context/WalletProvider/NewWalletViews/routes/WalletConnectV2Routes.tsx

```diff
@@ -24,7 +22,6 @@ export const NewWalletConnectV2Connect = () => {
   const navigate = useNavigate()
   const [loading, setLoading] = useState(false)
   const [error, setError] = useState<string | null>(null)
-  const isWcDirectConnectionEnabled = useFeatureFlag('WcDirectConnection')
```

Comment:
Can we remove flag across the board? This was added so we could have rows in a dumb hacked way under the WC pair button for PoC purposes, but if we're consuming it here, no need to keep flag anymore. 

<img width="1576" height="321" alt="Image" src="https://github.com/user-attachments/assets/42085b07-66bb-4d62-b4ac-17232a78b353" />